ros_ws/src/autonomous/neuralnetwork/configs/init/latest.conf

```diff
@@ -0,0 +1,36 @@
+FANN_FLO_2.1
+num_layers=3
+learning_rate=0.700000
+connection_rate=1.000000
+network_type=0
+learning_momentum=0.000000
+training_algorithm=2
+train_error_function=1
+train_stop_function=0
+cascade_output_change_fraction=0.010000
+quickprop_decay=-0.000100
+quickprop_mu=1.750000
+rprop_increase_factor=1.200000
+rprop_decrease_factor=0.500000
+rprop_delta_min=0.000000
+rprop_delta_max=50.000000
+rprop_delta_zero=0.100000
```

Comment:
Please check if you can remove the learning-related parameters since they will not be used.

Reply:
the save function isnt implemented by me, so i dont thinks its possible